tests/scripts/perf-sanity/benchmark-serve-prerelease.sh

```
@@ -0,0 +1,93 @@
+output_folder=benchmark.run.${SLURM_JOB_ID}.${start_time}.${commit_id}.${skip_pattern}.${select_pattern}
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+# the docker user is root, otherwise it can not write logs to this folder when running in scratch
+mkdir -p ${output_folder} && chmod 777 ${output_folder}
+
+run_benchmark() {
+    # Run benchmark on 8-GPU node using Docker
+    docker run --rm --ipc=host --ulimit memlock=-1 --ulimit stack=67108864 \
+        --gpus all \
+        -v /home/scratch.trt_llm_data:/home/scratch.trt_llm_data:ro \
+        -v `pwd`:`pwd` \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)